RInAppMessaging/Classes/InAppMessagingModule.swift

```diff
@@ -99,6 +100,14 @@ extension InAppMessagingModule {
             self.campaignsListManager.refreshList()
         }.perform()
     }
+
+    func shouldShowCampaignMessage(title: String, contexts: [String]) -> Bool {
+        guard let delegate = delegate else {
+            return true
+        }
+        return delegate.inAppMessagingShouldShowCampaignsWithContexts(contexts: contexts,
```

Comment:
Do you need a respondsToSelector check here? You've checked the delegate exists but we don't know if the delegate method was implemented.

Reply:
This method is not available for Swift objects. I think all swift protocols are marked as `required` in ObjC

Reply:
Ok, I thought because of the `@objc` it would be available but I now see the delegate extends `AnyObject`.